apps/cowswap-frontend/src/modules/limitOrders/pure/DeadlineSelector/deadlines.ts

```diff
@@ -6,7 +6,7 @@ export interface LimitOrderDeadline {
 }
 
 export const MIN_CUSTOM_DEADLINE = ms`30min`
-export const MAX_CUSTOM_DEADLINE = Math.round(ms`1y` / 2) // 6 months
+export const MAX_CUSTOM_DEADLINE = ms`182d` + ms`12h` // 6 months, matching backend's https://github.com/cowprotocol/infrastructure/blob/901ed8e2fe3ea57956585f107bdd7539c2e7d3d1/services/Pulumi.yaml#L15
```

Comment:
thanks so much for adding the link!!!

Reply:
weird the 12h thing, u know why they did that?

Reply:
Martin [did not mention](https://cowservices.slack.com/archives/C0361CDD1FZ/p1703863595046959)